prisma/migrations/20240207033957_init/migration.sql

```diff
@@ -90,5 +90,8 @@ CREATE TABLE "Import" (
     CONSTRAINT "Import_areaId_fkey" FOREIGN KEY ("areaId") REFERENCES "OfflineArea" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
```

Comment:
Because we haven't released this, I thought it was okay to generate a completely new migration. Let me know if you want me to add a new migration instead.

Reply:
technically this is deployed on some devices already but we've always been very clear about it being an experimental feature that's prone to major changes between pre-release versions. I don't anticipate integrating newer versions of this onto the devices where it's been deployed to, so I think what you've done is totally fine IMO